### ISSUE TITLE
API Level

### ISSUE THREAD

**Issue by narendra-bandineni** [AnyChart/AnyChart-Android#208]
I want to use AnyChart for Android api level <19 , Is there any way I can achieve that?

**Comment by Shestac92** [comment]
@narendra-bandineni 
We can't guarantee that the AnyChart library will work correctly on the API level <19, you can use it at your own risk.

**Comment by narendra-bandineni** [comment]
@Shestac92  Will this be available in future ?

**Comment by Shestac92** [comment]
@narendra-bandineni 
No, we are not planning to support the API level lower than 19.
